images/win/scripts/ImageHelpers/InstallHelpers.ps1

```diff
@@ -217,20 +206,20 @@ function Install-VsixExtension
         [switch] $InstallOnly
     )
 
-    if (!$InstallOnly)
-    {
-        $FilePath = Start-DownloadWithRetry -Url $Url -Name $Name
-    }
+    if (-not $InstallOnly)
+        {
+            $FilePath = Start-DownloadWithRetry -Url $Url -Name $Name
+        }
 
-    $ArgumentList = ('/quiet', "`"$FilePath`"")
+    $argumentList = ('/quiet', "`"$FilePath`"")
```

Comment:
Why did you refactor this variable to lower case here but keep it as ArgumentList everywhere else? I would suggest to leave $ArgumentList, it will reduce the amout of refactoring noise in PR

Reply:
Because, in `Install-VsixExtension` `argumentList` is function variable, and in other places (in helpers, of course) `ArgumentList` is function parameter.